packages/@aws-cdk-containers/ecs-service-extensions/test/integ.multiple-environments.expected.json

```diff
@@ -815,15 +815,15 @@
     "developmentenvironmentvpcPublicSubnet3NATGatewayAD65DE0B": {
       "Type": "AWS::EC2::NatGateway",
       "Properties": {
+        "SubnetId": {
+          "Ref": "developmentenvironmentvpcPublicSubnet3Subnet00B8A77C"
+        },
         "AllocationId": {
           "Fn::GetAtt": [
             "developmentenvironmentvpcPublicSubnet3EIPE2E366D3",
             "AllocationId"
           ]
         },
-        "SubnetId": {
```

Comment:
This reshuffling of json makes me sad. Anyone have any idea why this happens?

Reply:
not sure exactly, since `--dry-run` auto-updates these, its hard to tell which ones would fail for Asset hashes being different and which would just be reshuffling